### ISSUE TITLE
Add cart to primary navigation menu on mobile devices

### ISSUE THREAD

**Issue by davidsherlock** [easydigitaldownloads/themedd#62]
On AffiliateWP and various other sites based off Themedd, the cart shows in the primary navigation menu on mobile devices. I think we should do the same here, as this is expected behaviour I believe.

**Comment by amdrew** [comment]
This is actually possible, provided you create a menu and assign it to the mobile menu. I wanted Themedd to have its own mobile menu so you can offer a cut down, and not so confusing menu to mobile users.

However, there is a slight issue I'm aware of whereby if you assign your primary menu to both the primary menu position, as well as the mobile position, the cart icon does not show.

![screen shot 2017-06-27 at 12 35 47 pm](https://user-images.githubusercontent.com/52581/27565879-3bfb45d2-5b35-11e7-8b06-e64266c919eb.png)

**Comment by amdrew** [comment]
Ok this is ready for more testing.

The nav cart will now always be appended to the mobile menu, even if the nav cart has been moved to the primary navigation via the included filter, from the secondary navigation.

I also improved the logic so if no mobile menu is defined, the primary menu is used for the mobile menu. This means zero set up for the customer to get the mobile menu displaying. And of course, if the customer does want a new menu for mobile, they can easily do that by creating a menu and assigning it to the mobile menu location.
